### PR TITLE
Copy wsclient filters

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

File: pkg/rpcclient/wsclient.go [hunk 1]
```diff
@@ -678,7 +679,7 @@ func (c *WSClient) ReceiveBlocks(flt *neorpc.BlockFilter, rcvr chan<- *block.Blo
 	}
 	params := []interface{}{"block_added"}
 	if flt != nil {
-		params = append(params, *flt)
+		params = append(params, *flt.Copy())
```
Comment (AnnaShaleva) [inline pkg/rpcclient/wsclient.go:682]: I think it's not enough. Three lines below we add the same user's filter to the receiver, and after that client uses receiver's filter to filter out events. Thus, the user can still change something in filter and break the receiver.Let's provide copy to the receiver as far.

File: pkg/rpcclient/wsclient.go [hunk 2]
```diff
@@ -695,7 +696,8 @@ func (c *WSClient) ReceiveBlocks(flt *neorpc.BlockFilter, rcvr chan<- *block.Blo
 func (c *WSClient) SubscribeForNewTransactions(sender *util.Uint160, signer *util.Uint160) (string, error) {
 	var flt interface{}
 	if sender != nil || signer != nil {
-		flt = neorpc.TxFilter{Sender: sender, Signer: signer}
+		var f = neorpc.TxFilter{Sender: sender, Signer: signer}
+		flt = *f.Copy()
```
Comment (AnnaShaleva) [inline pkg/rpcclient/wsclient.go:700]: This one is OK because we use `flt` as a receiver's filter.

**Review by AnnaShaleva — COMMENTED** [review]
A nice one.

File: pkg/rpcclient/wsclient.go (hunk 1 above)
Comment (roman-khimov) [reply pkg/rpcclient/wsclient.go]: Fixed!

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**